WooCommerce/Classes/Extensions/UIImage+Woo.swift

```diff
@@ -399,7 +399,7 @@ extension UIImage {
 
 
     static var syncImage: UIImage {
-        return UIImage.gridicon(.sync).imageFlippedForRightToLeftLayoutDirection()
+        return UIImage(imageLiteralResourceName: "icon-sync-dot")
```

Comment:
not super sure if the design looks the same, but we used to have a gift icon with a dot on the top right corner using code. feel free to use the assets if there are design requirements that are harder to achieve with code though!

https://github.com/woocommerce/woocommerce-ios/blob/e81fdc73468c2ed43a4ec7a3cfc4b6b3df888cf6/WooCommerce/Classes/Extensions/UIImage+Woo.swift#L172-L183

light | dark
-- | --
![Simulator Screen Shot - iPhone 11 - 2020-07-01 at 21 36 43](https://user-images.githubusercontent.com/1945542/86251029-edcfa480-bbe3-11ea-8f63-d2467587e90f.png) | ![Simulator Screen Shot - iPhone 11 - 2020-07-01 at 21 36 56](https://user-images.githubusercontent.com/1945542/86250883-bd880600-bbe3-11ea-861b-e1166871cdf0.png)

Reply:
Wow I didn't know about that function 🙇 

Unfortunately(if I understand things correctly), that code does not change the icon colors from light to dark, when the user changes rendering mode while the app/screen is active. :-(